README.md

```diff
@@ -39,3 +39,30 @@ jobs:
 ## Notes
 
 Add executable permissions to scripts so that they are runnable after checkout `git update-index --chmod=+x script.sh`
+
+## CircleCI orb
+
+### Running orb in a workflow
+
+In the following example, Github version info can be read from `workspace/version.txt` after the job `generate-version` completes.
+
+```yaml
+version: 2.1
+description: This is a test job
+
+orbs:
+  github: qlikmats/github-version@0.0.13
+
+jobs:    
+  generate-version:
+    docker:
+    - image: circleci/golang:1.15
+    steps:
+      - checkout
+      - github/set-version
+
+workflows:
+  "Build my project":
+    jobs:
+      - generate-version
+```
```

Comment:
Should this be placed in `org-version/README.md` instead?

Reply:
yeah